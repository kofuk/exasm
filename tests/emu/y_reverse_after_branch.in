lui r0, 1
lli r1, 1
lli r2, 2
sw r1, (r0)
sw r2, (r0)
beqz r3, -6
nop
