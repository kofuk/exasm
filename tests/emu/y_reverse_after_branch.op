n
n
n
n
n
n
n
n
rn
rn
n
n
