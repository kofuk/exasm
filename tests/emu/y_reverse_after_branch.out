0x00
0x01
